arithmetization/src/main/java/net/consensys/linea/zktracer/module/exp/Exp.java

```diff
@@ -21,7 +21,7 @@
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import net.consensys.linea.zktracer.ColumnHeader;
-import net.consensys.linea.zktracer.container.stacked.set.StackedSet;
+import net.consensys.linea.zktracer.container.stacked.StackedSet;
```

Comment:
why can't we move to a `StackedList` here but in other modules we can?

Reply:
StackedSet <=> we want to kill duplicate and it's not ordered (as in MOD, ADD, WCP, etc ...)
StackedList <=> ordered list of unique operation (RLP_TXN, MMU, etc)